string topic_name
---
string topic_last_message